presentation-api/controlling-ua/startNewPresentation_displaynotallowed-manual.html

```diff
@@ -21,11 +24,19 @@
     // -------------------------------------------
     // Start New Presentation Test (error) - begin
     // -------------------------------------------
-    var startPresentation = function () {
+    presentBtn.onclick = function () {
         presentBtn.disabled = true;
         promise_test(function (t) {
             var request = new PresentationRequest(presentationUrls);
-            return promise_rejects(t, 'NotAllowedError', request.start());
+
+            // terminate the presentation connection when the presentation is started by accident
+            var connection;
+            t.add_cleanup(function() {
+                if(connection)
+                    connection.terminate();
+            })
```

Comment:
Just for the sake of code consistency, do you mean adding back the missing ";" here?